app/scripts/components/dashboard/charts-container.jsx

```diff
@@ -12,7 +12,7 @@ const ChartsContainer = React.createClass({
     return (
       <div className="charts-container">
         <div className="charts-container-header">
-          <T k="charts.toggle.activate" />
+          <span className="tab-label"><T k="charts.toggle.activate" /></span>
```

Comment:
@esoliani do you think we should let `<T>` take a `className` prop and put it on the `<span>` that holds the text? Or should we keep it simple and wrap it in a span like this when we want class names?